File: pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int cents=get_int("Change owed:");
     while(cents<=0)
     {
            printf("Please enter more than 0\n");
            cents=get_int("Change owed:");
     }

     int quarter=cents/25;
     cents=cents%25;

     int dimes=cents/10;
     cents=cents%10;

     int nickels=cents/5;
     cents=cents%5;

     int pennies=cents/1;

     int coins=quarter+dimes+nickels+pennies;
     printf("%i\n",coins);
     return 0;
}

File: Pset6/Cash/Cash.py
dollars = 0
cents = 0
coins = 0
while True:
    try:
        dollars = float(input("Change owed: "))
    except ValueError:
        print("Only floats.")

    else:
        cents = int(dollars*100)
        if cents >=0:
            break

quarters = cents//25
coins+=quarters
cents-=(25*quarters)

dimes = cents//10
coins+=dimes
cents-=(10*dimes)

nickels = cents//5
coins+=nickels
cents-=(5*nickels)

pennies = cents//1
coins+=pennies
cents-=(1*pennies)

print(coins)







